sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -162,6 +162,14 @@ const alwaysBeDeletedRule = "alwaysbedeletedrule";
         }
 
         should.equal(response.statusCode, 409, "Error must not be undefined");
+        should.equal(
+          response.message.startsWith("The messaging entity") ||
+            response.message.startsWith("Entity") ||
+            response.message.startsWith("SubCode") ||
+            response.message.startsWith("No service is hosted at the specified address."),
```

Comment:
This tells us that creating a non existing entity results in 4 different types of error messages. Are the 4 different types of messages based on the entity type or any other scenario?

@bterlson, would you consider the client library to step in here and provide a consistent error message or we throw whatever the service decided to throw?

Reply:
I don't want to second guess the service here, personally, but could be convinced otherwise

Reply:
The service sends us error information that could grammatically / structurally be constructed in any of those 4 known ways I would say.. (Shortened the last message string to convey this intent.)

We had specifically discussed about trying best to not take a hard dependency on error message sentences and that intent of testing error messages were mainly for verifying the error structure parsing - hence the tests were updated accordingly.